### PR TITLE
Fixed lesson and quiz icons in edit and saved mode

### PR DESCRIPTION
**Pull request by docia**
- [x] displayed proper lesson and quiz icons when the course is edited and when it is saved. 

### Before: 
Screenshot 2024-07-03 at 23 56 57

### After:
Screenshot 2024-07-03 at 23 51 13

Screenshot 2024-07-04 at 10 27 30

### CODE REVIEW

File: src/components/app-text-field/AppTextField.tsx [hunk 1]
```diff
@@ -5,11 +5,13 @@ import Tooltip from '@mui/material/Tooltip'
 
 import { styles } from '~/components/app-text-field/AppTextField.styles'
 import { TypographyVariantEnum } from '~/types'
+import { SxProps } from '@mui/material'
```
Comment (abalanovsky) [inline src/components/app-text-field/AppTextField.tsx:8]: move it upper, to other mui imports

Comment (docia) [reply src/components/app-text-field/AppTextField.tsx]: moved it

**Review by abalanovsky — APPROVED** [review]

**Review by Olenka-Hryk — APPROVED** [review]

**Comment by Olenka-Hryk** [comment]
I didn't investigate, but I'm just curious: will this icon with a placeholder be displayed if a user adds an attachment (for example, a PDF file)? Or is this functionality with attachments something we haven't developed yet?
Screenshot 2024-07-05 at 00 20 19

File: src/components/app-text-field/AppTextField.tsx [hunk 2]
```diff
@@ -10,6 +11,7 @@ interface AppTextFieldProps
   extends Omit<TextFieldProps, 'error' | 'helperText'> {
   errorMsg?: string
   withHelperText?: boolean
+  sx?: SxProps
```
Comment (YaroslavChuiko) [inline src/components/app-text-field/AppTextField.tsx:14]: I think line 14 is redundant, `TextFieldProps` already has `sx` prop

**Review by YaroslavChuiko — APPROVED** [review]

**Pull request merged**